test/LaunchDarkly.OpenFeature.ServerProvider.Tests/EvalContextConverterTests.cs

```diff
@@ -32,10 +32,9 @@ public void ItCanHandleBuiltInAttributes()
                 .Set("country", "country")
                 .Set("anonymous", true).Build();
 
-            var convertedUser = _converter.ToLdUser(evaluationContext);
+            var convertedUser = _converter.ToLdContext(evaluationContext);
```

Comment:
I left these tests mostly intact using User, and then added new a couple new tests.